### PR TITLE
First notebook deployment

### PR DESCRIPTION
**Pull request by ianfore**
Two basic Jupyter notebooks plus common module
Restructure of directories - one directory per language/tool. Another folder for R could appear.
Follows pattern used for NIH ODSS FHIR Training.

### CODE REVIEW

**Comment by ianfore** [comment]
Third notebook added with Dataframe of Observations

File: postman/dbGaP_FHIR_API.postman_collection.json
Comment (RadixSeven) [inline postman/dbGaP_FHIR_API.postman_collection.json:1]: There are links to these files in the readme. I'll check out the branch and fix them.

**Review by RadixSeven — APPROVED** [review]
I didn't run the notebooks myself. But quickly browsing them, they look OK. I updated the Postman links.

**Comment by ianfore** [comment]
We should seek to get an independent run of some kind.

**Pull request merged**